engine/node/score-node-api/src/main/java/io/cloudslang/engine/node/entities/WorkerNode.java

```diff
@@ -244,52 +247,84 @@ public void setWorkerRecoveryVersion(String workerRecoveryVersion) {
         this.workerRecoveryVersion = workerRecoveryVersion;
     }
 
-    @Override
+	@Override
+	public String getVersion() {
+		return version;
+	}
+
+	public void setVersion(String version) {
+		this.version = version;
+	}
+
+	@Override
 	public boolean equals(Object o) {
 		if (this == o) return true;
-		if (!(o instanceof WorkerNode)) return false;
+		if (o == null || getClass() != o.getClass()) return false;
 
 		WorkerNode that = (WorkerNode) o;
 
-		return new EqualsBuilder()
-				.append(this.uuid, that.uuid)
-				.append(this.description, that.description)
-				.append(this.hostName, that.hostName)
-				.append(this.installPath, that.installPath)
-				.append(this.active, that.active)
-				.append(this.ackTime, that.ackTime)
-                .append(this.ackVersion, that.ackVersion)
-				.append(this.os, that.os)
-				.append(this.jvm, that.jvm)
-				.append(this.dotNetVersion, that.dotNetVersion)
-				.append(this.groups, that.groups)
-                .append(this.bulkNumber, that.bulkNumber)
-                .append(this.workerRecoveryVersion, that.workerRecoveryVersion)
-				.isEquals();
+		if (active != that.active) return false;
```

Comment:
Like the EqualsBuilder better, is it a matter of taste, or  that you found an issue with that.